lib/scenic/adapters/postgres.rb

```diff
@@ -17,6 +17,10 @@ def self.drop_view(name)
         execute "DROP VIEW #{name};"
       end
 
+      def self.extensions(base = ActiveRecord::Base)
```

Comment:
Ah, ok. Makes sense. Thanks.

I', really trying to get this merged in but struggling with some intermittently failing tests on travis. Might have to merge this before I get that resolved. Hoping to have a release out tomorrow at some point.